src/dashboard/tests/test_ingest.py

```diff
@@ -76,3 +77,187 @@ def test_add_metadata_files_view(self):
             ["<h1>", "  Add metadata files<br />", "  <small>test</small>", "</h1>"]
         )
         assert title in response.content
+
+
+def _assert_file_node_properties_match_record(file_node, record):
+    assert file_node["type"] == "file"
+    assert file_node["id"] == record["fileuuid"]
+    # the relative path of the node is encoded
+    assert file_node["relative_path"].decode("base64") == record["relative_path"]
+    # the node title is the encoded file name
+    assert (
+        file_node["title"].decode("base64") == record["relative_path"].rsplit("/", 1)[1]
```

Comment:
Would `os.path.sep` be better to replace `/` or even, `os.path.basename()` satisfy what you're doing here?

Reply:
Ah thanks!